src/terra-dev-site/contributing/DeveloperWorkflow.c.contributing.md

```diff
@@ -0,0 +1,171 @@
+
+* Our ESLint config is based on [Airbnb’s JavaScript style guide](https://github.com/airbnb/javascript).
+* Our Stylelint config is based on [sass-guidelin.es](https://sass-guidelin.es/).
+
+Inside the monorepos, running the following command in the root directory to run both ESLint and Stylelint for all packages.
+
+`npm run lint`
+
+* You can also run `npm run lint:js` and this will only run ESLint.
+* You can also run `npm run lint:scss` and this will only run StyleLint.
+
+Along with being able to run these scripts across the entire project, you can run these scripts in individual package directories.
+
+Change directories to the specific package in your terminal, e.g. `cd packages/terra-button`, and run the same lint commands for ESLint and Stylelint.
+
+```
+npm run lint
+npm run lint:js
+npm run lint:scss
+```
+
+We recommend that you install a plugin to your editor to run [ESLint](https://eslint.org/docs/user-guide/integrations#editors) and [Stylelint](https://stylelint.io/user-guide/complementary-tools/#editor-plugins) automatically which will then show you any errors inline.
+
+## Testing Components
+
+Terra UI has 2 types of tests:
+* Unit tests, using Jest
+* Browser tests, using Webdriver.io
+
+### Jest
+We use Jest for testing, including unit tests for functionality and snapshot testing for components.
+
+If you make intentional changes to an existing component, you will need to update its snapshot:
+npm test -- -u
+
+At minimum, all components should have a "does it render" test:
+
+```jsx
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Link from './Link';
+
+it('renders without crashing', () => {
+  const div = document.createElement('div');
+  ReactDOM.render(<Link />, div);
+});
+```
+
+
+You may also want to test class level functions:
+
+```jsx
+it('tracks the number of clicks', () => {
+  const div = document.createElement('div');
+  const instance = ReactDOM.render(<Link />, div);
+
+  t.equals(instace.state.clicks, 0);
+  instance.trackClick();
+  t.equals(instace.state.clicks, 1);
+});
+```
+
+Be careful when adding tests; we need to be wary of when we're effectively testing the React internals as opposed to our own components.
+
+#### Running Jest Tests
+
+// TODO
+
+### Webdriver.io (WDIO)
+
+// TODO
+
+#### Visual Regression Testing
+
+// TODO
+
+#### Running Webdriver Tests
+
+// TODO
+
+### A Note On Testing
+
+Note that the test suite needs to pass for your changes to be accepted, so it's worth running this locally during development and before committing.
+
+Even though we'd like to automate as much as possible, a comprehensive manual test is never a bad idea, especially if you're working in an area of the codebase that's particularly business critical. Reviewers are also encouraged to put UI changes through their paces, to act as a last line of defense before merging.
+
+## Updating The Changelog
+
+// TODO
```

Comment:
This is the link I send people for Changelog info: https://keepachangelog.com/en/1.0.0/